File: datasets/IR-Plag/intellij-original-kotlin-rest-java/case-05/plagiarized/L3/05/method.java
package IR;

import java.util.*;

class method{
	//method reverse
		public static void beautyReverse(int num) 
	{
		while (num != 0)
			{
			int r = num % 10;
				System.out.print(r);
			num = num / 10;
		}
		System.out.println();
	}
	
	//prog utama
	public static void main(String[] args) 
	{
			System.out.print("Enter an integer: ");
			Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		//pamggil method
			beautyReverse(n);
	}
	
	

}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L6/05/method.kt
package IR

import java.util.*

internal object method {
    //method reverse
    var input: Scanner = Scanner(System.`in`)
    fun beautyReverse(num: Int) {
        var num = num
        while (num > 0) {
            print(num % 10)

            num /= 100 * 10
        }
        println()
    }

    fun reverse() {
        val n = input.nextInt()
        beautyReverse(n)
    }

    //prog utama
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")


        //pamggil method
        reverse()
    }
}